docker.mk

```diff
@@ -51,7 +51,7 @@ clean: docker.clean
 docker.clean:
 	rm -rf .build
 
-docker.test.shard: $(foreach image,$(shell echo $(images) | python util/balancecontainers.py $(SHARDS) | awk 'NR%$(SHARDS)==$(SHARD)'),$(docker_test)$(image))
+docker.test.shard: $(foreach image,$(shell echo $(images) | tr ' ' '\n' | awk 'NR%$(SHARDS)==$(SHARD)'),$(docker_test)$(image))
```

Comment:
This isn't related to an error message fix.  Please remove OR update the PR to describe the purpose of the change.,